recipes/default.rb

```diff
@@ -16,3 +16,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+repo = 'qemu-ev'
+
+# Install Virt SIG gpg repo key
+package 'centos-release-virt-common' do
+  only_if { node['kernel']['machine'] == 'x86_64' }
+end
+
+yum_repository repo do
+  to_hash.keys.each do |k|
+    send(k, node['yum'][repo][k]) if node['yum'][repo][k]
```

Comment:
I would to `k.to_s` here since `to_hash` will return symbols instead of strings. Just to be safe.